File: backend/src/main/kotlin/com/casadosreclamos/dto/FinishingDto.kt
package com.casadosreclamos.dto

import com.casadosreclamos.model.request.Finishing

data class FinishingDto(var id: Long?, var name: String?, var obsolete: Boolean?) {
    constructor(finishing: Finishing) : this(finishing.id, finishing.name, finishing.obsolete)
}

File: frontend/src/models/FinishingDto.ts
export default class FinishingDto {
  id: number;
  name: string;
  obsolete: boolean;

  constructor(obj: FinishingDto) {
    this.id = obj.id;
    this.name = obj.name;
    this.obsolete = obj.obsolete;
  }
}
